components/x-article-save-button/src/ArticleSaveButton.scss

```diff
@@ -0,0 +1,35 @@
+@import 'o-icons/main';
+@import 'o-colors/main';
+@import 'o-typography/main';
+
+$icon-size: 38px;
+
+@mixin background-icon($name, $color) {
+	@include oIconsGetIcon($name, oColorsGetPaletteColor($color), $icon-size, $iconset-version: 1);
+}
+
+.root {
+	display: inline-block;
+}
+
+.button {
+	@include background-icon(bookmark-outline, #000000);
```

Comment:
Oh, can you use `oColorsGetPaletteColor(#000000)`? I would expect to see `black` being passed in here.